StereoKitC/stereokit.h

```diff
@@ -2134,6 +2137,9 @@ SK_API void*             backend_android_get_jni_env  ();
 SK_API backend_graphics_ backend_graphics_get           ();
 SK_API void             *backend_d3d11_get_d3d_device   ();
 SK_API void             *backend_d3d11_get_d3d_context  ();
+SK_API void             *backend_d3d11_get_d3d_deferred_context();
+SK_API void             *backend_d3d11_get_d3d_deferred_mtx();
+SK_API uint32_t         backend_d3d11_get_d3d_main_thread_id();
```

Comment:
I'm a little hesitant about these names, I think! In this case, the name is intended to be a hint as to what the `void*` is, so `d3d_device` is a `ID3D11Device`, and `d3d_context` is a `ID3D11DeviceContext`.

In that light, I think the names may be a bit better like this?
`backend_d3d11_get_deferred_d3d_context`
`backend_d3d11_get_deferred_mtx`
`backend_d3d11_get_main_thread_id`

Reply:
Alternatively:
`d3d_context_deferred`
`mtx_deferred`

Reply:
Hinting the type sounds helpful. I think I prefer to have deferred before like your first suggestions since it reads a little better. Good for either but I'll go ahead and update it